extensions/amp-user-notification/amp-user-notification.md

```diff
@@ -0,0 +1,101 @@
+<!--
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-user-notification"></a> `amp-user-notification`
```

Comment:
@cramforce do you mind review this? still thinking through styling section (need to do a couple of examples to grasp it fully)